handlers/settings.go

```diff
@@ -0,0 +1,62 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/gaia-pipeline/gaia/services"
+	"github.com/gaia-pipeline/gaia/workers/pipeline"
+
+	"github.com/gaia-pipeline/gaia"
+	"github.com/labstack/echo"
+)
+
+// SettingsPollOn turn on polling functionality.
+func SettingsPollOn(c echo.Context) error {
+	storeService, _ := services.StorageService()
+	gaia.Cfg.Poll = true
+	err := pipeline.StartPoller()
+	if err != nil {
+		c.String(http.StatusBadRequest, err.Error())
+	}
+	configStore := &gaia.StoreConfig{}
```

Comment:
We should load it in case we have future settings and we don't want to overwrite them here.